src/components/periodpicker/periodpicker.ts

```diff
@@ -0,0 +1,90 @@
+import { PluginObject } from 'vue';
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+import { ModulVue } from '../../utils/vue/vue';
+import { PERIODPICKER_NAME } from '../component-names';
+import DatepickerPlugin, { DatePickerSupportedTypes } from '../datepicker/datepicker';
+import WithRender from './periodpicker.html';
+
+export class MDateRange {
+    from: DatePickerSupportedTypes;
+    to: DatePickerSupportedTypes;
+}
+
+@WithRender
+@Component
+export class MPeriodPicker extends ModulVue {
+    @Prop()
+    value: MDateRange;
+
+    @Prop()
+    min: DatePickerSupportedTypes;
+
+    @Prop()
+    max: DatePickerSupportedTypes;
+
+    toIsFocused: boolean = false;
+
+    get firstInputState(): any {
+        return {
+            props: { label: 'Du', value: this.internalValue.from, min: this.min, max: this.max },
```

Comment:
'Du' en français? Peux-tu utiliser i18n?

Reply:
Version préliminaire du periodpicker, sera attaqué dans une itération future